packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -312,54 +313,6 @@ export const DefaultSummaryConfiguration: ISummaryConfiguration = {
 	nonRuntimeHeuristicThreshold: 20,
 };
 
-export interface IGCRuntimeOptions {
```

Comment:
yesss you reduced this file size by almost 1.5%

Reply:
Haha. Well this actually moved so the other file size increased :D